<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestor de Base de Datos</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font: italic 20pt &quot;David&quot;;
text-decoration: underline;
}</string>
    </property>
    <property name="text">
     <string>Gestor de Base de Datos</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>601</width>
      <height>261</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Create DB</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 20pt &quot;David&quot;;
text-decoration: underline;</string>
      </property>
      <property name="text">
       <string>Crear Base de Datos</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 20pt &quot;David&quot;;</string>
      </property>
      <property name="text">
       <string>create databases </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_DBnew">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_bdnew">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Create Table</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 20pt &quot;David&quot;;
text-decoration: underline;</string>
      </property>
      <property name="text">
       <string>Crear Tabla</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 20pt &quot;David&quot;;</string>
      </property>
      <property name="text">
       <string>CREATE TABLE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_tablenew">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_tablenew">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>150</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_useDB">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 20pt &quot;David&quot;;</string>
      </property>
      <property name="text">
       <string>USE DATABASES</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Insert</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 20pt &quot;David&quot;;
text-decoration: underline;</string>
      </property>
      <property name="text">
       <string>Insertar Campos a la Tabla</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_insert1">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Insertar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 20pt &quot;David&quot;;</string>
      </property>
      <property name="text">
       <string>Tama√±o</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_tipo1">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Int</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Varchar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Float</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Date</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 20pt &quot;David&quot;;</string>
      </property>
      <property name="text">
       <string>Tipo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tam1">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>120</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_nameC1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 20pt &quot;David&quot;;</string>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_insert2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>160</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Insertar</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_tipo2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>160</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Int</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Varchar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Float</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Date</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="tam2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>160</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_nameC2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Values</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 20pt &quot;David&quot;;
text-decoration: underline;</string>
      </property>
      <property name="text">
       <string>Insertar Valores a la Tabla</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 20pt &quot;David&quot;;</string>
      </property>
      <property name="text">
       <string>Insertar into</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 20pt &quot;David&quot;;</string>
      </property>
      <property name="text">
       <string>Values</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_valuescamp">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_valuesdata">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Insertar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Consultas</string>
     </attribute>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 20pt &quot;David&quot;;</string>
      </property>
      <property name="text">
       <string>SELECT</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_select">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 20pt &quot;David&quot;;</string>
      </property>
      <property name="text">
       <string>From</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_from">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 20pt &quot;David&quot;;</string>
      </property>
      <property name="text">
       <string>where</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_where">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>120</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Consultar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
